components/kyma-environment-broker/internal/runtime/components/components.go

```diff
@@ -12,5 +12,6 @@ const (
 	BackupInit              = "backup-init"
 	AvSBridge               = "avs-bridge"
 	Eventing                = "eventing"
+	Connectivity            = "connectivity"
```

Comment:
where is it used?

Reply:
do we have a component Connectivity in Kyma or maybe we will have?

Reply:
In this PR it's not used yet. We plan to create the Connectivity component and pass the Connectivity binding data to it probably by overrides. Do you think we should delete it now and add it in the next step or it could just be there?

Reply:
I'd leave it, since it will be installed as a new component in the runtime.